app/bundles/CoreBundle/Event/CustomTemplateEvent.php

```diff
@@ -63,11 +63,9 @@ public function getVars()
     }
 
     /**
-     * @param arrat $vars
-     *
-     * @return CustomTemplateEvent
+     * @param array<string, mixed> $vars
      */
-    public function setVars($vars)
+    public function setVars(array $vars): CustomTemplateEvent
```

Comment:
Adding types to existing properties or return types in public or protected methods is considered a BC break.  Please improve the types in the method comments only.

Reply:
Thanks! I've fixed it